### DISCUSSION TITLE
Is this still in development?

### DISCUSSION THREAD

**Discussion by ledcomp** [Jyers/Marlin#1423] [General Discussion]
First of all, this firmware made a huge difference in the operation of my printer, so thank you for all of the hard work.

I was curious if this is still being developed since it was merged with the main branch? If so, is there any idea when the next update might be released?

Thanks again for the awesome work.

**Comment by martinlmtl** [comment]
I was asking myself the same question and afraid to ask. As far as I can see, it's doing more than a month we got no update on github as well as seeing any message from jyers.

**Comment by ledcomp** [comment]
Well that's good to hear.

Thanks
